README.md

```diff
@@ -1,18 +1,29 @@
-# [nem2-open-api-generator](https://github.com/NEMStudios/nem2-open-api-generator)
-Project in charge of generating the nem2-sdk clients using openapi cli, gradle and npm.
+# [symbol-openapi-generator](https://github.com/nemtech/symbol-openapi-generator)
 
-The generate script will create and install different flavours of java and javascript rest clients according to the current NEM Open API specification.
+[![Build Status](https://travis-ci.org/nemtech/symbol-openapi-generator.svg?branch=master)](https://travis-ci.org/nemtech/symbol-openapi-generator)
 
-The official NEM Api descriptor can be found [here](https://github.com/nemtech/nem2-docs/blob/master/source/resources/collections/openapi3.yaml):
+Project in charge of generating the API client libraries used by the NEM2-SDKs.
 
-# Requirements
+The ``generate`` script creates different flavours of clients according to the [current OpenAPI specification](https://github.com/nemtech/nem2-openapi).
+
+## Supported languages
+| Language   | Flavours          | 
+|------------|------------------|
+| Javascript | typescript-node  |
+| Java       | vertx            |
+|            | jersey2          |
+|            | okhttp-gson      |
+
+See possible options for languages and flavours [here](https://openapi-generator.tech/docs/generators/).
+
+## Requirements
 
 * NPM 8
-* Java 8 - One way of installing is using [sdkman](https://sdkman.io/)
+* Java 8
 
-# How to use it
+## Usage
 
-Once tools have been installed, just run
+Once you have installed the dependencies, run:
```

Comment:
To install the dependencies, run:

Reply:
the dependencies are NPM and Java listed obove